packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -1007,23 +1007,35 @@ export class AnalyzerService {
             }
         };
 
+        const seenDirs = new Set<string>();
+        const visitDirectory = (absolutePath: string, includeRegExp: RegExp) => {
+            const realDirPath = this._fs.realpathSync(absolutePath);
+            if (seenDirs.has(realDirPath)) {
+                this._console.warn(`Skipping recursive symlink "${absolutePath}" -> "${realDirPath}"`);
```

Comment:
I don't have to keep this if we want to just silently ignore these problems. Won't show by default anyway, more useful for debugging.